.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
I'd like to continue testing against CV2, since that's what all the current linux distributions include.